.github/workflows/action-failure.yml

```diff
@@ -0,0 +1,22 @@
+name: Action failure notification
+
+on: 
+  workflow_call:
+    inputs:
+      environment:
+        required: true
+        type: string
+
+jobs:
+  notifications:
+    name: Failure notifications
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Slack Notification
+        uses: rtCamp/action-slack-notify@v2
+        env:
+          SLACK_CHANNEL: planx-notifications-internal
+          SLACK_USERNAME: PlanX Action Failure
```

Comment:
nit: since this is already an internal notification channel, is "Github Actions" a more meaningful username here - eg it more immediately points us to the source?

Reply:
This is a good shout - I'll make this change 👍